### PR TITLE
Place tools/index.hbs text into .ftl file

### PR DESCRIPTION
**Pull request by eignnx**
Part of #798 .

I used `nav-tools` for the page title since that text is already defined in [`common.ftl`](https://github.com/rust-lang/www.rust-lang.org/blob/i18n/templates/fluent-resource/en-US/common.ftl).

Lines in `tools.ftl` have been wrapped at 80 characters.

My naming scheme for localization variable names has been:
* `...-header` for headers/titles (ex: `tools-editor-support-heading`)
* `...-description` for body text
* `...-link` for link/button text

### CODE REVIEW

File: templates/fluent-resource/en-US/tools.ftl [hunk 1]
```diff
@@ -1,4 +1,52 @@
-tools-write-ide-prose = Whether you prefer working with code from the command line, or using
-        rich graphical editors, there’s a Rust integration available for your
-        editor of choice. Or you can build your own using the
+# index.hbs
+tools-editor-support-heading = First-class editor support
+tools-write-ide-description = Whether you prefer working with code from the
```
Comment (Manishearth) [inline templates/fluent-resource/en-US/tools.ftl]: update the name of this id in `xx-AU` too, please

File: templates/fluent-resource/en-US/tools.ftl [hunk 2]
```diff
@@ -1,4 +1,52 @@
-tools-write-ide-prose = Whether you prefer working with code from the command line, or using
-        rich graphical editors, there’s a Rust integration available for your
-        editor of choice. Or you can build your own using the
+# index.hbs
```
Comment (Manishearth) [inline templates/fluent-resource/en-US/tools.ftl]: tools/index.hbs

File: templates/fluent-resource/en-US/tools.ftl [hunk 3]
```diff
@@ -1,4 +1,52 @@
...
+        Iterator.”</i> <br> Clippy helps developers of all experience levels
+        write idiomatic code, and enforce standards.
+tools-automation-clippy-link = Go to repo
+
+tools-automation-cargo-doc-heading = Cargo Doc
+tools-automation-cargo-doc-description = Cargo’s doc builder makes it so no API
+        ever goes undocumented. It’s available locally through
+        <code class="nowrap">cargo doc</code>, and online for public crates
+        through <a href="https://docs.rs">docs.rs</a>.
+tools-automation-cargo-doc-link = Go to site
```
Comment (Manishearth) [inline templates/fluent-resource/en-US/tools.ftl]: new line at EOF please

File: templates/fluent-resource/en-US/tools.ftl (hunk 1 above)
Comment (eignnx) [reply templates/fluent-resource/en-US/tools.ftl]: Sure thing! Do you mind if I rename it (in both files) to `tools-editor-support-description` for consistency?

Comment (Manishearth) [reply templates/fluent-resource/en-US/tools.ftl]: go for it

**Comment by Manishearth** [comment]
Thanks!

**Pull request merged**